services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerClusterConfig.java

```diff
@@ -649,6 +653,14 @@ public VeniceControllerClusterConfig(VeniceProperties props) {
     } else {
       this.persistenceType = PersistenceType.IN_MEMORY;
     }
+
+    if (props.containsKey(DEFAULT_CONCURRENT_PUSH_DETECTION_STRATEGY)) {
+      this.concurrentPushDetectionStrategy =
+          ConcurrentPushDetectionStrategy.valueOf(props.getString(DEFAULT_CONCURRENT_PUSH_DETECTION_STRATEGY));
+    } else {
+      this.concurrentPushDetectionStrategy = ConcurrentPushDetectionStrategy.DUAL;
```

Comment:
Usually, it's better to keep the current mode as default and then flip the switch in tests and rollout config gradually to allow safe config/feature rollbacks without version downgrades IMHO

Reply:
The changes between default and dual is very little, this will save us pain for rolling out another config

Reply:
If it brakes the flow due to unseen edge case, rolling back version is much bigger pain than config rollback. That said if we're confident about the change, let's keep it

Reply:
Yeah, I plan to keep it in dual mode as its not really adding new risky code path.